api/client/trust_lookup_experimental.go

```diff
@@ -0,0 +1,14 @@
+// +build experimental
+
+package client
+
+import "github.com/docker/docker/registry"
+
+// trust_should_lookup checks whether the trust system should be used to
+// lookup a reference for an image. In this experimental version, it returns
+// true if trust is enabled and we are not already doing a pull by digest,
+// and also returns true if it is an official image being that is not
+// already being pulled by digest
+func trust_should_lookup(byDigest, official bool) bool {
```

Comment:
ugh, yeah, was naming the files with Docker's existing naming scheme and fell back into Python syntax. Will fix now